### PR TITLE
Add code to get correct application URL when running on IBM Cloud systems

### PR DESCRIPTION
**Pull request by groeges**
Signed-off-by: Steven Groeger <groeges@uk.ibm.com>

### CODE REVIEW

File: cmd/utils.go [hunk 1]
```diff
@@ -1686,13 +1686,54 @@ func KubeDelete(log *LoggingConfig, fileToApply string, namespace string, dryrun
 	return kerr
 }
 
+//KubeGetNodePortURLIBMCloud issues several kubectl commands and prints the concatenated URL
+func KubeGetNodePortURLIBMCloud(log *LoggingConfig, service string, namespace string, dryrun bool) (url string, err error) {
+	kargs := append([]string{"pod"})
```
Comment (EnriqueL8) [inline cmd/utils.go]: you can remove the append() wrapping `[]string{"pod"}`

Comment (groeges) [reply cmd/utils.go]: Sure will do that.

Comment (groeges) [reply cmd/utils.go]: Done

**Comment by groeges** [comment]
@EnriqueL8 Had to rebase to pickup latest changes in master branch. 
Would you be able to re-approve this PR. Thanks.

**Pull request merged**